src/NodeApi/JSValue.cs

```diff
@@ -21,38 +20,56 @@ namespace Microsoft.JavaScript.NodeApi;
 
     internal JSRuntime Runtime => Scope.Runtime;
 
+    /// <summary>
+    /// Creates an empty instance of <see cref="JSValue" />, which implicitly converts to
+    /// <see cref="JSValue.Undefined" /> when used in any scope.
+    /// </summary>
     public JSValue() { }
 
-    public JSValue(napi_value handle) : this(handle, JSValueScope.Current)
```

Comment:
This constructor was quite convenient in a number of cases. What is the advantage to remove it?

Reply:
It was only used in one place: the `operator JSValue(napi_value)` in this class. I removed it thinking it might be better to be explicit when using `JSValueScope.Current`, since in this PR I had to fix many places that were incorrectly using `JSValueScope.Current` instead of the scope on the provided `JSValue`. But as long as we still have that operator that is using the current scope implicitly (which we use everywhere) then this constructor is no worse. So I've added it back in the update.